ADApp/pluginSrc/NDFileHDF5.cpp

```diff
@@ -392,25 +392,29 @@ asynStatus NDFileHDF5::createTree(hdf5::Group* root, hid_t h5handle)
     }
 
     // Write contant datasets to file
-    this->writeHdfConstDatasets(new_group, root);
-
-    getIntegerParam(NDFileHDF5_storeAttributes, &storeAttributes);
-    if (storeAttributes == 1){
-      // Set some attributes on the group
-      this->writeHdfAttributes(new_group,  root);
-    }
+//    this->writeHdfConstDatasets(new_group, root);
```

Comment:
With writeHdfConstDatasets() commented out at this point, this function is not used anywhere else. Is it no longer required and can be deleted? It should not be left in the code like this if it is not used...

I must admit I've lost track a bit in these many changes so I don't have the overview to determine if this function can go...